src/axom/core/examples/core_acceleration.cpp

```diff
@@ -135,18 +136,27 @@ void demoAxomExecution()
     C[i] = 0;
   }
 
-  axom::for_all<axom::CUDA_EXEC<256>>(
+  #if defined(__CUDACC__)
```

Comment:
Do we want to hard-code the thread-block size values?  Have we thought of making it more flexible to let users choose?